cli/src/cli.rs

```diff
@@ -2209,6 +2209,16 @@ pub(crate) fn build_balance_message(
 }
 
 pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, 'v> {
+    macro_rules! ACCOUNT_STRING {
+        () => {
+            r#"Can be:
+      * a bs58 pubkey string
+      * path to keypair file
```

Comment:
Probably should call out hardware wallet as well: https://docs.solana.com/wallet/cli-wallets/remote-wallet#specify-a-hardware-wallet-key
Perhaps:
`* keypair URL to hardware wallet` ?

Reply:
added